src/plugins/discover/public/application/components/data_grid/data_grid.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './_data_grid_table.scss';
```

Comment:
Does the css get imported twice or is it deduplicated? (not sure if the dedupe option is turned on for the loader)

Given this will be removed, feel free to ignore if this ends up having no real impact.

Reply:
You are right. The other one needs to go. Will add it to s subsequent PR.